packages/@aws-cdk/aws-eks/test/alb-controller.integ.snapshot/manifest.json

```diff
@@ -168,7 +168,10 @@
         "/aws-cdk-eks-cluster-alb-controller-test/KubectlLayer/Resource": [
           {
             "type": "aws:cdk:logicalId",
-            "data": "KubectlLayer600207B5"
+            "data": "KubectlLayer600207B5",
+            "trace": [
+              "!!DESTRUCTIVE_CHANGES: WILL_REPLACE"
```

Comment:
Is this a concern?

Reply:
No, a lambda layer is a stateless resource, and we are replacing it with an upgraded version of kubectl. So, the replacement is expected here.